ll-property-information-edit.html

```diff
@@ -262,12 +288,26 @@
           value: function() {
```

Comment:
Does anyone know if we have this data in an api? Or a shared location, because I know we use it across ui's